e2e/pages/match/occupancyViewPage.ts

```diff
@@ -0,0 +1,34 @@
+import { Page, expect } from '@playwright/test'
+import { Premises } from '@approved-premises/api'
+import { MatchBasePage } from './matchBasePage'
+import { E2EDatesOfPlacement } from '../../steps/assess'
+import { ReleaseTypeLabel } from '../../../server/utils/applications/releaseTypeUtils'
+
+export class OccupancyViewPage extends MatchBasePage {
+  static async initialize(page: Page, premisesName: Premises['name']) {
+    await expect(page.locator('h1')).toContainText(`View spaces in ${premisesName}`)
+
+    return new OccupancyViewPage(page)
+  }
+
+  async shouldShowMatchingDetails({
+    datesOfPlacement,
+    duration,
+    releaseType,
+  }: {
+    datesOfPlacement: E2EDatesOfPlacement
+    duration: string
+    releaseType: ReleaseTypeLabel
+  }): Promise<void> {
+    await super.shouldShowDatesOfPlacement(datesOfPlacement, duration)
```

Comment:
Again, I'm not sure about starting to use `super` instead of simply `this` -- not on a fundamental level, but for consistency, as it's not something we've used anywhere else.